### DISCUSSION TITLE
How to ignore specific folders within a monorepo?

### DISCUSSION THREAD

**Discussion by 0xjjpa** [vercel/vercel#6490] [Help]
Is it possible to ignore specific folders within a monorepo to avoid building packages that are not needed? e.g. if I have two dashboards, and PR is only doing changes to one, can I configure the project so it only builds if files are changed on a specific folder?

[Here's](https://github.com/hoprnet/hopr-community/pull/57) and example: I'm working on `hopr-stake` which is a package inside that monorepo. Another existing project/package called `dashboard` has been configured there. The linked PR only contains changes to `package/hopr-stake` but the build is being triggered for `package/dashboard`, which creates unnecessary noise in my PR.

Is it possible to only build `package/dashboard` when file changes are done in that folder? Is there a configuration file that allows me to tweak when and where to actually build which project? The alternative would be to create a custom GitHub workflow, but I rather stick to the native GitHub Vercel workflow if possible.

**Comment by leerob** [comment]
Can you try using `.vercelignore`? https://vercel.com/guides/prevent-uploading-sourcepaths-with-vercelignore

**Reply by 0xjjpa** [reply]
Unfortunately [it doesn't seem to have worked](https://github.com/hoprnet/hopr-community/pull/76). For reference,

```
root
\_ dashboard
\_ app
\_ .vercelignore
```
and the contents of `.vercelignore` are `app`, yet when I do a PR that changes something inside `app` (e.g. `app/package.json`, the Vercel bot gets triggered to build `dashboard`. You can see the [PR](https://github.com/hoprnet/hopr-community/pull/76), and how the Vercel created the preview/build link, despite being there no changes inside `dashboard/*`
